kitsune/wiki/tests/test_templates.py

```diff
@@ -389,18 +389,18 @@ def test_document_with_fallback_locale(self):
         """The document template falls back to fallback locale if there is
         custom wiki fallback mapping for the locale and the locale have no translation
         exists."""
-        # Create an English document and a bn-BD translated document
+        # Create an English document and a es translated document
```

Comment:
Instead of `es` here, let's use `bn` since we actually added it as a fallback for `bn-IN` and `bn-BD`.